h/templates/emails/reply_notification.html.jinja2

```diff
@@ -1,23 +1,14 @@
-<p>
-  <a href="{{ reply_user_url }}">{{ reply_user.username }}</a>
-  has just left a reply to your annotation on
-  &ldquo;<a href="{{ document_url }}">{{ document_title }}</a>&rdquo;.
-</p>
-<p>
-  On <a href="{{ parent_url }}">{{ parent.created | human_timestamp }}</a>
```

Comment:
Looks like the `human_timestamp` Jinja2 extension will be made unused by this change, so let's remove it.